src/main/java/com/example/weather/client/services/WeatherDataService.java

```diff
@@ -17,6 +21,7 @@
     private final WeatherDataRepo weatherDataRepo;
     private final WeatherClient weatherClient;
     private final WeatherDataMapper weatherDataMapper;
+    private final int maxPageSize = 100;
```

Comment:
constants in java are usually static
alternatively you can use `application.yml`